bindings/rust/bench/src/harness.rs

```diff
@@ -64,6 +64,25 @@ pub trait TlsBenchHarness: Sized {
 
     /// Get whether or negotiated version is TLS1.3
     fn negotiated_tls13(&self) -> bool;
+
+    /// Send data from one connection
```

Comment:
Not sure your current comments are helpful. Also note, if it's obvious what a function does, you don't need to comment it (peep the comment for get_negotiated_cipher_suite). It's only if you have extra information that would be useful to a reader that you should write a comment.
```suggestion
    /// Send application data
```

Reply:
I still want to keep the "from one connection" part to emphasize that data is being sent from one connection to another within the same `TlsBenchHarness`. However, I think that specifying the data as "application data" is very useful.